gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/LoadConfiguration.java

```diff
@@ -0,0 +1,381 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.bigquery;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import com.google.api.services.bigquery.model.JobConfigurationLoad;
+import com.google.common.base.MoreObjects;
+import com.google.common.collect.ImmutableList;
+import com.google.gcloud.bigquery.JobInfo.CreateDisposition;
+import com.google.gcloud.bigquery.JobInfo.WriteDisposition;
+
+import java.io.Serializable;
+import java.nio.channels.SeekableByteChannel;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Google BigQuery Configuration for a Load operation. A Load configuration can be used to build a
```

Comment:
Nit: should "Load" --> "load?  It looks like "load" isn't capitalized in the BigQuery docs (e.g. in step 2 [here](https://cloud.google.com/bigquery/loading-data-into-bigquery#loaddatagcs))